### DISCUSSION TITLE
Data Versioning JSON files with pysonDB

### DISCUSSION THREAD

**Discussion by kurianbenoy** [pysonDB/pysonDB#29] [Ideas]
**How would this feature be useful?**

I feel versioning json files in Database is going  a useful feature. I can see it useful for version
data artifacts in form of JSON objects.

**Describe the solution you'd like**
No ideas at the moment

**Describe alternatives you've considered**
- https://github.com/iterative/dvc

**Comment by Adwaith-Rajesh** [comment]
@kurianbenoy It's a simple DB., not a huge Data Science library. 😂

**Comment by fredysomy** [comment]
Well its worth a try..😅

**Comment by kurianbenoy** [comment]
I agree @Adwaith-Rajesh, this feature should never make the project unnecessarily complex. Yet in case of databases also, there is usually a concept of migrations, which track the schema changes over the time. In Django and Flask, you may have used it.

I was just thinking something on similar lines for versioning json data stored in the database.
